types/react-tracking/test/react-tracking-without-types-tests.tsx

```diff
@@ -13,6 +13,11 @@ function customEventReporter(data: { page?: string }) {}
 )
 class ClassPage extends React.Component<any> {
     @track({ event: 'Clicked' })
+    @track((_props, _state, [e]: [React.MouseEvent]) => {
```

Comment:
@drbalm - this file is supposed to test functionality without types.

Reply:
@damassi, *doh*, thanks for catching this! 😄 

I removed the typing in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40916/commits/3f373a780d6809d7808bd3e15b4247d7cb171e61